src/math-translator.js

```diff
@@ -229,6 +255,17 @@ function normalizeTranslatedMath(math, lang) {
          // We cannot allow a literal space here, cause Tex would ignore it
          {langs: MATH_RULES_LOCALES.THOUSAND_SEP_AS_THIN_SPACE,
             regex: /([0-9])\{?~\}?([0-9])(?=[0-9]{2})/g, replace: '$1\\,$2'},
+
+         // KaTeX seems to support more trig functions then default LaTeX
+         // but we're using operatorname for them as well.
+         // Also, they are currently blocked by linter.
+         // https://katex.org/?data=%7B%22displayMode%22%3Atrue%2C%22leqno%22%3Afalse%2C%22fleqn%22%3Afalse%2C%22throwOnError%22%3Atrue%2C%22errorColor%22%3A%22%23cc0000%22%2C%22strict%22%3A%22warn%22%2C%22trust%22%3Afalse%2C%22macros%22%3A%7B%22%5C%5Cf%22%3A%22f(%231)%22%7D%2C%22code%22%3A%22%25%20%5C%5Cf%20is%20defined%20as%20f(%231)%20using%20the%20macro%5Cn%5C%5Csin%20%5C%5Carcsin%20%5C%5Ccos%20%5C%5Carccos%20%5C%5C%5C%5C%20%5C%5Ctan%20%5C%5Carctan%20%5C%5Ctg%20%5C%5Carctg%20%5C%5C%5C%5C%5Cn%5C%5Ccot%20%5C%5Ccotg%20%5C%5Cctg%20%5C%5Coperatorname%7Barccot%7D%20%5C%5C%5C%5C%5Cn%5C%5Ccsc%20%5C%5Ccosec%20%5C%5Coperatorname%7Barccsc%7D%20%5C%5C%5C%5C%5Cn%5C%5Coperatorname%7Bsen%7D%7Bx%7D%20%5C%5Csin%7Bx%7D%20%5C%5Csin%20x%22%7D
+         {langs: MATH_RULES_LOCALES.TAN_AS_TG.concat(
+                 MATH_RULES_LOCALES.COT_AS_COTG,
+                 MATH_RULES_LOCALES.COT_AS_CTG,
+                 MATH_RULES_LOCALES.CSC_AS_COSEC),
+            regex: /\\(tg|arctg|cotg|ctg|cosec)/g,
+            replace: '\\operatorname{$1}'},
```

Comment:
Btw, why is this done here and not in `translateMathOperators`?

Reply:
`translateMathOperators`, called from `translateMath` is applied on the original En string, whereas normalizedTranslatedMath is applied the templated string translated by the translator. I tried to expand the jsdoc for `normalizeTranslatedMath` to make that a bit clearer.

Reply:
Thank you @danielhollas